app/templates/settings/password.hbs

```diff
@@ -1 +1 @@
-{{settings/password-section}}
\ No newline at end of file
+{{settings/password-section submit='changePassword' passwordCurrent=passwordCurrent passwordNew=passwordNew passwordRepeat=passwordRepeat isLoading=isLoading}}
```

Comment:
instead of `submit='changePassword'` pass in a proper action please. 

`submit=(action 'changePassword')`